### PR TITLE
Cleanup Paper Component

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

This PR updates the top padding for the `Paper` component and relocates the application of the padding to the component itself instead having to apply it every time it's used in other components.

In the process, a lot of components were wrapped in a `Paper` component when they didn't need to be so I cleaned up those files. I also moved the border styles from the custom `border` prop to Material UI's built-in variant instead.

## How to test

Check the app for any components that look like they have too much padding applied to them.

### CODE REVIEW

**Comment by bnussman** [comment]
Starting to review... Looks like on the Create Linode page, things are a bit tighter in general. Is this ok?

https://user-images.githubusercontent.com/6440455/119179161-99702600-ba3c-11eb-91ca-a4a6d3f57ead.mov

**Comment by bnussman** [comment]
Starting the review... I like the idea of using Material UI's built-in variant prop. I also like we are cleaning up the use of paper.

**Comment by bnussman** [comment]
Looks like on the Create Linode page, things are a bit tighter in general. Is this ok?

https://user-images.githubusercontent.com/6440455/119180311-22d42800-ba3e-11eb-9d75-54c04f5e1248.mov

File: packages/manager/src/features/linodes/LinodesDetail/LinodeBackup/LinodeBackup.tsx [hunk 1]
```diff
@@ -83,7 +83,6 @@ type ClassNames =
 const styles = (theme: Theme) =>
   createStyles({
     paper: {
-      padding: theme.spacing(3),
```
Comment (bnussman) [inline packages/manager/src/features/linodes/LinodesDetail/LinodeBackup/LinodeBackup.tsx:86]: Looks better if this line is kept.

**Comment by bnussman** [comment]
Charts now have more padding than before

https://user-images.githubusercontent.com/6440455/119190030-ce837500-ba4a-11eb-9a7b-4b1a9e0e4b39.mov

**Comment by bnussman** [comment]
The Popular posts now looks a bit odd. Looks like it needs a bit more top padding and less left padding
Screen Shot 2021-05-21 at 3 43 05 PM

Screen Shot 2021-05-21 at 3 43 12 PM

File: packages/manager/src/features/linodes/LinodesCreate/SelectLinodePanel.tsx [hunk 2]
```diff
@@ -80,56 +76,49 @@ class SelectLinodePanel extends React.Component<CombinedProps> {
...
+          count,
+          data: linodesData,
+          handlePageChange,
+          handlePageSizeChange,
+          page,
+          pageSize,
+        }) => {
+          return (
+            <>
+              <Paper className={`${classes.root}`} data-qa-select-linode-panel>
```
Comment (bnussman) [inline packages/manager/src/features/linodes/LinodesCreate/SelectLinodePanel.tsx:90]: `className={`${classes.root}`}` -> `className={classes.root}`

File: packages/manager/src/features/linodes/LinodesCreate/SelectBackupPanel.tsx [hunk 3]
```diff
@@ -131,31 +128,30 @@ class SelectBackupPanel extends React.Component<CombinedProps, State> {
 
     return (
       <Paper className={`${classes.root}`}>
```
Comment (bnussman) [inline packages/manager/src/features/linodes/LinodesCreate/SelectBackupPanel.tsx:130]: className={classes.root}

**Comment by tiffwong** [comment]
@bnussman thanks for the thorough review! I adjusted the left padding of the Help & Support page but reducing the top padding is actually intentional 😅

**Review by bnussman — APPROVED** [review]

**Review by DevDW — APPROVED** [review]
Looked through the app and didn't observe anything that was off 🚢

**Pull request merged**